themes/lucky/lucky.theme.sh

```diff
@@ -0,0 +1,59 @@
+#! bash oh-my-bash.module
+
+_omb_deprecate_const 20000 red "$_omb_prompt_brown" "${_omb_deprecate_msg_please_use/'%s'/_omb_prompt_brown}"
+_omb_deprecate_const 20000 blue "$_omb_prompt_navy" "${_omb_deprecate_msg_please_use/'%s'/_omb_prompt_navy}"
+_omb_deprecate_const 20000 green "$_omb_prompt_green" "${_omb_deprecate_msg_please_use/'%s'/_omb_prompt_green}"
+_omb_deprecate_const 20000 purple "$_omb_prompt_purple" "${_omb_deprecate_msg_please_use/'%s'/_omb_prompt_purple}"
+
+SCM_GIT_SHOW_MINIMAL_INFO=true
+SCM_THEME_PROMPT_PREFIX=""
+SCM_THEME_PROMPT_SUFFIX=""
+
+L_BLUE=$_omb_prompt_navy
+L_CYAN=$_omb_prompt_teal
+L_GREEN=$_omb_prompt_green
+L_NORMAL=$_omb_prompt_normal
+L_PURPLE=$_omb_prompt_purple
+L_RED=$_omb_prompt_brown
+L_WHITE=$_omb_prompt_white
+
+L_BOLD_RAD=$_omb_prompt_bold_brown
+L_BOLD_YELLOW=$_omb_prompt_bold_olive
+
+L_PREFIX=$SCM_THEME_PROMPT_PREFIX
+L_SUFFIX=$SCM_THEME_PROMPT_SUFFIX
+
+# reset git_prompt_minimal_info
+function git_prompt_minimal_info {
+  local ref
+  local status
+  local git_status_flags=('--porcelain')
+  SCM_STATE=${L_GREEN}${SCM_THEME_PROMPT_CLEAN}
```

Comment:
```suggestion
  local SCM_STATE=${L_GREEN}${SCM_THEME_PROMPT_CLEAN}
```

Unless you want to expose this variable to the user or the caller functions so that they can directly reference the variable content, please localize it. This variable is used only in this function. The original function defined in `lib/omb-prompt-base.sh` intentionally exposed the variables to the caller, but it doesn't apply to the present case. You might think unlocalized variables don't affect the main shell because this function is only used inside a subshell, but in the Oh-My-Bash codebase, functions should be designed so that they don't leave unnecessary side effects.